app/graphql/mutations/create_edit_individual_event.rb

```diff
@@ -0,0 +1,28 @@
+module Mutations
+  class CreateEditIndividualEvent < BaseMutation
+    description "Create a new individual event for a logged-in user"
+
+    null true
+
+    argument :input, Types::Input::CreateEditIndividualEventInputType, required: true
+
+    def resolve(input:)
```

Comment:
test? what happens if `IndividualEvent.find()` fails?

Reply:
I believe it's going to lead to a 500 server side error.

Reply:
I added a case for this so we now get a nice error

```
{
  "errors": [
    {
      "message": "Couldn't find IndividualEvent with 'id'=999"
    }
  ]
}
```